core-bundle/src/EventListener/MakeResponsePrivateListener.php

```diff
@@ -53,7 +53,7 @@ public function __invoke(ResponseEvent $event): void
         $response = $event->getResponse();
 
         // Disable the default Symfony auto cache control
-        $response->headers->set(AbstractSessionListener::NO_AUTO_CACHE_CONTROL_HEADER, true);
+        $response->headers->set(AbstractSessionListener::NO_AUTO_CACHE_CONTROL_HEADER, '1');
```

Comment:
Why `'1'`? FoS uses `1` here: https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/blob/master/src/EventListener/UserContextListener.php#L169

Reply:
:man_shrugging:  Because the `HeaderBag` expects values to be `string|string[]`.

see https://github.com/symfony/symfony/blob/73cb33a54e25e24b62844f971791e9765002422e/src/Symfony/Component/HttpFoundation/HeaderBag.php#L137